src/locales/en.json

```diff
@@ -177,19 +177,12 @@
   },
   "cross_device": {
     "intro": {
-      "document": {
-        "title": "Continue verification on your mobile",
-        "take_photos": "We'll walk you through taking the photos",
-        "action": "Continue"
-      },
-      "face": {
-        "title": "Continue face verification on your mobile",
-        "take_photos": "Complete face verification to prove it's really you",
-        "action": "Continue"
-      },
-      "sub_title": "It's easy, fast and secure.",
-      "sms": "We'll SMS a secure link to your mobile (no app download required)",
-      "return_computer": "Return to your computer to complete your verification"
+      "title": "Continue on your phone",
+      "sub_title": "Here's how to do it:",
+      "description_li_1": "Send a secure link to your phone",
```

Comment:
Is the word "send" appropriate when we talk about scanning a QR code? Maybe we should say  "Get a secure link on to your phone" or something like that.

Reply:
Good point. It would also cover when a user copies the link since we're not really sending a link for that scenario also. @raemarielee thoughts?

Reply:
I think it's OK to stick with "send" in this instance – at this stage we want the user to focus on the fact that they'll be directed to their phone from their desktop. The "Get a secure link" button then brings them more detailed actions ("scan code" / "send SMS" / "copy & paste link").